config/webpack.config.prod.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const webpack = require('webpack')
+
+module.exports = {
+  output: {
+    filename: 'app.[hash].min.js'
+  },
+  devtool: '#cheap-module-source-map',
+  plugins: [
+    new webpack.optimize.DedupePlugin(),
+    new webpack.optimize.OccurenceOrderPlugin(),
+    new webpack.optimize.UglifyJsPlugin({
+      mangle: true,
+      compress: {
+        warnings: false
+      }
+    }),
+    new webpack.DefinePlugin({
+      __SERVER__: false,
+      __DEVELOPMENT__: false,
+      __DEVTOOLS__: false,
+      __STACK_ASSETS__: true
```

Comment:
Ok @gregorylegarec 
I guess it worth to keep compliance if it is needed for configuration updates.